Framework/Backend/log/InfoLoggerReceiver.js

```diff
@@ -17,7 +17,7 @@ module.exports = class InfoLoggerReceiver extends EventEmitter {
    */
   constructor() {
     super();
-
+    this.log = new Log('InfoLoggerReceiver');
```

Comment:
I would have put Log instance otherwise like:
`const log = new require('./Log.js')('InfoLoggerReceiver');`

To avoid pollution of namespace `this.log =` and so the use of Log is also the same for non-class files like index.js
it is also compatible with the current code, less modifications and better readability (imo).
What do you think?

Reply:
Here is an example from expressjs:

https://github.com/expressjs/express/blob/master/lib/router/route.js#L16

Reply:
I think it's a good idea. I'll make changes to the PR.